### DISCUSSION TITLE
hidden analytics?

### DISCUSSION THREAD

**Discussion by hhannis** [Eugeny/tabby#5184] [Q&A] [answered]
"Enable analytics" is off/disabled, still Tabby is sending data to sentry.io. Expected?

**Comment by Eugeny** [comment, marked as answer]
Yes - only crash tracebacks are sent to Sentry, with no identifying information. The "Enable analytics" checkbox enables analytics data (OS type and app version) to be send to Mixpanel.

**Comment by lawweiliang** [comment]
@hhannis   
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
